### PR TITLE
Issue 7

### PR DESCRIPTION
**Pull request by ajcpwnz**
Added ability to create and view a list of teams

### CODE REVIEW

File: app/controllers/dashboard/teams_controller.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    before_action :require_auth
+
+    def index
+    end
+
+    def new
+    end
+
+    def create
+      t = current_user.teams.create(team_params)
```
Comment (IlyaUmanets) [inline app/controllers/dashboard/teams_controller.rb]: `team`

File: app/controllers/dashboard/teams_controller.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    def team_params
+      params.require(:team).permit(:name, :color, :owner_id)
+    end
+
+    def email_batch
+      params[:email_batch].split(', ')
+    end
+
+    def manage_invites(emails, team_id)
```
Comment (IlyaUmanets) [inline app/controllers/dashboard/teams_controller.rb]: move this to service object

File: app/views/layouts/mailer.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (IlyaUmanets) [inline app/views/layouts/mailer.html.erb]: Use slim

File: config/application.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Add configuration values here, as shown below.
+#
+# pusher_app_id: "2954"
```
Comment (IlyaUmanets) [inline config/application.yml]: You have to create `application.example.yml`. And left values blank to secure data

File: spec/actions/dashboard/teams_spec.rb [hunk 5]
```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+feature 'Visit "new team" page' do
+  let(:user) { create(:user) }
+
+  scenario 'as guest' do
+    visit new_dashboard_team_path
+    expect(page).to have_text 'Sorry, but you need to authenticate to access this page.'
```
Comment (IlyaUmanets) [inline spec/actions/dashboard/teams_spec.rb]: You should use locale for static text

File: app/views/layouts/mailer.html.erb (hunk 3 above)
Comment (ajcpwnz) [inline app/views/layouts/mailer.html.erb]: For some reason github doesnt see that i changed file :)

**Pull request merged**